tests/test_updater_ng.py

```diff
@@ -17,20 +17,22 @@
 from securesystemslib.interface import import_rsa_privatekey_from_file
 from securesystemslib.signer import SSlibSigner
 
-import tuf.unittest_toolbox as unittest_toolbox
 from tests import utils
-from tuf import exceptions, ngclient
+from tuf import exceptions, ngclient, unittest_toolbox
 from tuf.api.metadata import Metadata, TargetFile
 
 logger = logging.getLogger(__name__)
 
 
 class TestUpdater(unittest_toolbox.Modified_TestCase):
+    """Test the Updater class from tuf.ngclient.updater.py"""
```

Comment:
I'm not sure that the '.py' is needed. Add between simple quotes (?).

Reply:
Yea, well it's doesn't really matter, I can change it, but I am not sure if there is some benefit in doing so.
I think I have used it in other places as well.

Reply:
Indeed. I think my comment was more about having a single standard for consistency.